sqlserver/hatch.toml

```diff
@@ -12,7 +12,7 @@ setup = ["single", "ha"]
 # time out. until we're able to modify and parallelize the work we'll limit the per-driver tests to only a single
 # sqlserver version
 [[envs.default.matrix]]
-python = ["3.9"]
+python = ["3.11"]
```

Comment:
I need a bit more time to verify the upgrade for `sqlserver` as one of the package `pyro4` does not officially support python 3.11 according to [readme](https://github.com/irmen/Pyro4#development-halted-switch-to-pyro5-instead)

Reply:
`pyro4` is removed